From dc08e56acc5513341550b3d1a26b8d877e0f0456 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Wed, 5 Sep 2012 16:45:18 +0800
Subject: [PATCH 2014/2463] ENGR00223103 ESAI ASRC: clean build warning

BUILD WARNING:
WARNING: arch/arm/mach-mx6/built-in.o(.data+0x7e44): Section mismatch in
reference from the variable sab_audio_data to the (unknown reference)
.init.rodata:(unknown) The variable sab_audio_data references the
(unknown reference) __initconst (unknown) If the reference is valid then
annotate the variable with __init* or __refdata (see linux/init.h) or
name the variable:
*_template, *_timer, *_sht, *_ops, *_probe, *_probe_one, *_console

In this patch, remove esai_p2p struct with init attribute.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_arm2.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_arm2.c b/arch/arm/mach-mx6/board-mx6q_arm2.c
index 29911a2..81612e5 100644
--- a/arch/arm/mach-mx6/board-mx6q_arm2.c
+++ b/arch/arm/mach-mx6/board-mx6q_arm2.c
@@ -1562,7 +1562,7 @@ static const struct pm_platform_data mx6_arm2_pm_data __initconst = {
 	.suspend_exit	= arm2_suspend_exit,
 };
 
-static const struct asrc_p2p_params esai_p2p __initconst = {
+static const struct asrc_p2p_params esai_p2p = {
        .p2p_rate = 44100,
        .p2p_width = ASRC_WIDTH_24_BIT,
 };
-- 
1.7.7.4

